src/server/game/World/World.cpp

```diff
@@ -1184,6 +1184,13 @@ void World::LoadConfigSettings(bool reload)
     m_bool_configs[CONFIG_OFFHAND_CHECK_AT_SPELL_UNLEARN]            = sConfigMgr->GetOption<bool>("OffhandCheckAtSpellUnlearn", true);
     m_int_configs[CONFIG_CREATURE_STOP_FOR_PLAYER]                   = sConfigMgr->GetOption<uint32>("Creature.MovingStopTimeForPlayer", 3 * MINUTE * IN_MILLISECONDS);
 
+    m_float_configs[CONFIG_WATER_BREATH_TIMER]                       = sConfigMgr->GetOption<float>("WaterBreath.Timer", 3.0f);
+    if (m_float_configs[CONFIG_WATER_BREATH_TIMER] <= 0)
+    {
+        LOG_ERROR("server.loading", "WaterBreath.Timer ({}) must be > 0. Using 3 instead.", m_float_configs[CONFIG_WATER_BREATH_TIMER]);
+        m_float_configs[CONFIG_WATER_BREATH_TIMER] = 3;
```

Comment:
any reason to use float and not int? since it's cast to int later on anyways?

Reply:
It is giving more flexibility. For example, wanting 1.5 minutes. It is only cast to int after times by Minutes and Milliseconds, so fractions of minutes are supported.

Reply:
I have changed this to now use milliseconds directly and avoid the cast, as Kitzunu suggest.